apps/ui/src/components/Ui/InputString.vue

```diff
@@ -14,18 +14,25 @@ const props = defineProps<{
   definition: any;
 }>();
 
-const dirty = ref(false);
+const isModified = ref(false);
+
+// Mark as dirty if there's an error and the value differs from default
+const isDirty = computed(
+  () =>
+    isModified.value ||
+    (!!props.error && model.value !== (props.definition.default ?? ''))
```

Comment:
We can remove props.error check there (dirty is more of a flag to say if it was modified).

`UiWrapperInput` later on checks both `error` and `dirty` to decide if it should show error.

Reply:
@ChaituVR was this updated? It still shows up like this in my diff on Github 🤔

Reply:
hmm yes, what do you see? 🤔 

maybe browser cache
<img width="1113" height="993" alt="image" src="https://github.com/user-attachments/assets/5fc88a24-d48f-416e-a449-7bf89004ffd7" />

Reply:
Now I see the same as yours.